get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
+	} catch ( e ) {
+		console.error(
+			'Could not read in the required configuration file.\n' +
+			'This file should exist as `config.js` inside the project root directory.\n' +
+			'Please consult the project README.md for further information.\n'
+		);
+		process.exit( 0 );
```

Comment:
I should put a comment in there, or fail with an exit code.

Although I started with `process.exit( 1 );` it was spewing out a bunch of `npm` errors because of the exit code and the explanation then got lost. This was a pragmatic compromise to make sure the message was clear and visible.

Thoughts?